### PR TITLE
Show an ordered list of forms from a form creator's groups

### PR DESCRIPTION
**Pull request by chao-xian**
## What problem does this pull request solve?

Trello card: https://trello.com/c/0ObcUSir

The list of forms was appearing as they are returned by the API which doesn't make sense to the user. This PR applies alphabetical sorting.

## Before
localhost_3000_groups_NBgwE3D4

## After
localhost_3000_groups_NBgwE3D4 (1)

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: app/presenters/form_list_presenter.rb
Comment (lfdebrux) [inline app/presenters/form_list_presenter.rb:46]: What happens if there is more than one form with the same name?

**Review by lfdebrux — APPROVED** [review]
Thanks for doing this, this is a really nice example of what idiomatic Rails code should look like; I especially appreciate the detailed commit messages.

I have one comment/question, but otherwise I think this is good to go after a rebase.

File: app/presenters/form_list_presenter.rb
Comment (alice-carr) [reply app/presenters/form_list_presenter.rb]: I've changed this so that now if two forms have the same name, we then sort on `created_at`

**Review by lfdebrux — COMMENTED** [review]
Thanks for sorting the sorting @alice-carr! 😁 

One more thing, I can see there's a merge commit in the branch now, I think we prefer to rebase branches on main before merging so each PR has only one merge commit in the main branch.

Are you happy to do the rebase?

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**
